### PR TITLE
[receiver/azuremonitorreceiver] Added new attributes for metrics

### PR DESCRIPTION
**Pull request by altuner**
**Component:** 
receiver/azuremonitor

**Description:** 
Added new attributes: name, type and resource group to the metrics

**Link to tracking Issue:** n/a

**Testing:** tested on existing envs

**Documentation:** n/a

### CODE REVIEW

**Review by mwear — COMMENTED** [review]
This looks good to me. I do think it's worthy of a changelog. Would you mind adding one describing this enhancement?

**Review by mwear — APPROVED** [review]

**Comment by mwear** [comment]
@MovieStoreGuy, would you mind taking a look at this when you get a chance? I would not expect a review from @codeboten since he is on vacation.

**Review by codeboten — APPROVED** [review]
please resolve the conflicts and we can get this merged

File: Makefile.Common [hunk 1]
```diff
@@ -165,7 +165,7 @@ fmt: $(GOIMPORTS)
 
 .PHONY: lint
 lint: $(LINT) checklicense misspell
-	$(LINT) run --allow-parallel-runners --verbose --build-tags integration --path-prefix $(shell basename "$(CURDIR)")
+	$(LINT) run --allow-parallel-runners --verbose --build-tags integration --timeout=30m --path-prefix $(shell basename "$(CURDIR)")
```
Comment (codeboten) [inline Makefile.Common:168]: This change appears unrelated

**Review by codeboten — COMMENTED** [review]
I can re-run the job. See https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/25875 for the linter timing out
